cms/templates/cms/home_page.html

```diff
@@ -117,14 +114,12 @@ <h5 class="upper-case">Cost Savings</h5>
       <div class="quote-section">
         <div class="row quote">
           <img class="person" src="{% static 'images/reif1.png' %}" alt="">
-          <h2 class="quote-text">"We will find people who never thought they would be able to apply to MIT.
-          And they will discover that they are much stronger in the global competition than they think."</h2>
+          <h2 class="quote-text">"We will find people who never thought they would be able to apply to MIT. And they will discover that they are much stronger in the global competition than they think."</h2>
```

Comment:
Should these be curly quotes, “like this”? Or is it fine to leave them as straight quotes, "like this"?